apps/ledger-live-mobile/src/screens/Market/SearchHeader.tsx

```diff
@@ -27,11 +28,7 @@ function SearchHeader({ search, refresh }: Props) {
         limit: 20,
       });
     }
-  }, [debouncedSearch, refresh, search]);
-
-  useEffect(() => {
-    setInputSearch(search);
-  }, [setInputSearch, search]);
+  }, [debouncedSearch, refresh, search, setInputSearch]);
```

Comment:
I think that you don't need setInputSearch. In latest version it's not required (don't know if we use the correct version of React and linter), can you check, please?
example in the doc:
https://react.dev/reference/react/useEffect#connecting-to-an-external-system

Reply:
Yes indeed